test-cases/gutenberg/group.md

```diff
@@ -0,0 +1,355 @@
+# Group Block - Test Cases
+
+--------------------------------------------------------------------------------
+
+##### TC001
+
+### Group block is visible in Inserter menu
+
+-   CLick `AppenderButton ( + )`
+-   Check if the `Group` block is visible
+-   Check if you are able to press and add `Group` block
+
+--------------------------------------------------------------------------------
+
+##### TC002
+
+### Group block after add is selected, empty and renders AppenderButton inside
+
+-   Add a `Group` block 
+-   Check if `Group` block is selected
+-   Check if `Group` block is empty and shows only `AppenderButton`
+
+Expected look:
+ <<<SCREENSHOT>>
+  
+--------------------------------------------------------------------------------
+
+##### TC003
+
+### AppenderButton is render properly
+
+-   Add a `Group` block 
+-   Check if `AppenderButton` renders properly
+-   Check if `AppenderButton` is align with the content
+
+Expected look:
+ <<<SCREENSHOT>>
+
+--------------------------------------------------------------------------------
+
+##### TC004
+
+### Group AppenderButton allows to open Inserter and add new block to group
+
+-   Add a `Group` block 
+-   Check if you are able to open `Inserter` menu by press on `AppenderButton` inside `Group`
+-   Check if the block is added inside the `Group` before the `AppenderButton`
+-   Select a `Group` block once again
+-   Check if you are able to add another block
+-   Check if the block is added inside the `Group` before the `AppenderButton`
+
+--------------------------------------------------------------------------------
+
+##### TC005
+
+### DefaultAppender allows to open Inserter and add block outside the group
```

Comment:
I have add this because there was some regression when changing the appender/separator logic which affects the behaviour of Default Appender

I thought it is nice to check if  the DefaulAppender works as well

Reply:
Should we delete this one ?

Reply:
sorry I had missed this one. yes I think we should better remove. it is a pretty intuitive functionality and one will be able to notice if sth is odd while doing other tests

Reply:
ok, removed